blocksync/pool.go

```diff
@@ -30,9 +32,34 @@ eg, L = latency = 0.1s
 */
 
 const (
-	requestIntervalMS         = 2
-	maxPendingRequestsPerPeer = 20
-	requestRetrySeconds       = 45
+	requestIntervalMS = 2
+	// Request retry seconds is calculated dynamically,
+	// this is used when we don't have enough samples
+	defaultRetrySeconds = 45
+
+	// Border values for dynamic retry timer calculation
+	// Small blocks (1 KB) retry after 5 seconds
+	// Large blocks (20 MB+) retry after 60 seconds
+	minBlockSizeBytes = 1024             // 1 KB
+	maxBlockSizeBytes = 20 * 1024 * 1024 // 20 MB
+	minRetrySeconds   = 5
+	maxRetrySeconds   = 60
+
+	// Border values for dynamic maxPendingRequestsPerPeer
+	// Small blocks can have more concurrent requests
+	// Large blocks should have fewer to avoid bandwidth saturation
+	maxPendingForSmallBlocks = 40
```

Comment:
```suggestion
	// maxPendingForSmallBlocks Border values for dynamic maxPendingRequestsPerPeer
	// Small blocks can have more concurrent requests
	// Large blocks should have fewer to avoid bandwidth saturation
	maxPendingForSmallBlocks = 40
```

just to follow standard go docs